README.md

```diff
@@ -49,6 +49,12 @@ To run the tests, run the following command:
 poetry run pytest
 ```
 
+Install [pre-commit](https://pre-commit.com/) to format code before committing.
+
+```shell
+pip install pre-commit

```

Comment:
Also mention the command needed to install the hook?

```suggestion
pip install pre-commit
pre-commit install
```

Reply:
> `pre-commit` should be included as a development dependency in pyproject.toml.

Done